### PR TITLE
lib: Provides headers, operation name and variables to sidecar handler

### PR DESCRIPTION
**Pull request by soluchok**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -238,57 +232,36 @@ function InigoPlugin(config) {
...
-          query.ingest();
-        }
-      }
-    }
-
-    // If request is blocked
-    if (processed?.result.status == "BLOCKED") {
+    if (processed?.response != null) {
       requestContext.request = { http: requestContext.http };  // remove request from pipeline
-      requestContext.context.inigo.blocked = true; // set blocked state
```
Comment (adambenhassen) [inline index.js:274]: Restore this line, make sure it's defined where it meets the condition.

File: index.js [hunk 2]
```diff
@@ -238,57 +232,36 @@ function InigoPlugin(config) {
...
 
-    // Create request context, for storing blocked status
-    if (requestContext.context.inigo === undefined) {
-      requestContext.context.inigo = { blocked: false };
-    }
-
-    // is an introspection
-    if (processed?.request.data?.__schema != undefined) {
-      requestContext.request = { http: requestContext.http };  // remove request from pipeline
-      requestContext.context.inigo.blocked = true; // set blocked state
```
Comment (adambenhassen) [inline index.js:263]: Restore this line, make sure it's defined where it meets the condition.

File: index.js [hunk 3]
```diff
@@ -238,57 +232,36 @@ function InigoPlugin(config) {
...
+    });
 
     // Process request
-    const processed = query.processRequest(auth);
+    const processed = req.processRequest(requestContext.request.http.headers);
     requestContext.inigo = { processed };
 
-    // Create request context, for storing blocked status
-    if (requestContext.context.inigo === undefined) {
-      requestContext.context.inigo = { blocked: false };
```
Comment (adambenhassen) [inline index.js:257]: Restore this line, make sure it's defined where it meets the condition.

File: index.js (hunk 1 above)
Comment (soluchok) [reply index.js:274]: we do not have `blocked` logic anymore

Comment (adambenhassen) [reply index.js:274]: that part is public

**Comment by soluchok** [comment]
Closing pull request: commit has gone away

**Pull request closed without merging**